### ISSUE TITLE
Official image on Docker Hub

### ISSUE THREAD

**Issue by 205g0** [oven-sh/bun#1159]
### What is the problem this feature will solve?

I consider this more as a must-have than a feature. A well-maintained (=gets every release) image on Docker Hub with extensive env vars is crucial and has the lowest entry barrier. Maybe there's a reason you don't have this and I missed something but yeah. Besides, having just a Dockerfile isn't the same.

### What is the feature you are proposing to solve the problem?

Maintained image on Dcoker Hub

**Comment by xhyrom** [comment]
Isn't https://github.com/oven-sh/bun/pkgs/container/bun-obj what you want?

**Comment by 205g0** [comment]
Absolutely, thanks to hinting me to that image. I still would consider Docker Hub. Always when I want to try something new, e.g., database, language or whatever, I head to Docker image checking out their readme. A good image has two things:

- All relevant config options as env vars, so you can quickly try out test an image without fiddling around with proprietary config files and having these also well document right in the image's readme
-  Is getting updated with every release

So, you don't need Docker Hub for this, you can host this on Github as well. Then there should be a dedicated readme just for the image and the general bun js readme should point to that one at the top. On Docker Hub you would have download figures which can give some street credits if required.

**Comment by 205g0** [comment]
oh, just have seen, it's already mentioned on the main readme, so my bad, shall I close this issue or do you want to keep it open for a potential Docker Hub release? Whatever, feel free to close it

**Comment by Electroid** [comment]
We now publish Docker images to [`oven/bun`](https://hub.docker.com/r/oven/bun/). We also want to become an ["official"](https://github.com/docker-library/official-images/blob/master/NEW-IMAGE-CHECKLIST.md) image, but that will be done soon.

**Comment by Electroid** [comment]
Work being tracked here: https://github.com/docker-library/official-images/pull/14272

**Comment by jbergstroem** [comment]
Would love to see this happen. How can I help? I can update/create a new PR in the official-images repo?
